imports/plugins/core/router/client/initApollo.js

```diff
@@ -0,0 +1,26 @@
+import { ApolloClient } from 'apollo-client';
+import { ApolloLink } from 'apollo-link';
+import { HttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
+
+export const meteorAccountsLink = new ApolloLink((operation, forward) => {
+  const token = Accounts._storedLoginToken();
+
+  operation.setContext(() => ({
+    headers: {
+      'meteor-login-token': token,
+    },
+  }));
+
+  return forward(operation);
+});
+
+
+export default function initApollo() {
+  return new ApolloClient({
+    link: meteorAccountsLink.concat(new HttpLink({ uri: Meteor.absoluteUrl('graphql-alpha') })),
```

Comment:
If latest 1.14.0 is merged into this, `Meteor.absoluteUrl` can be changed to `Reaction.absoluteUrl`. (Everywhere else in app has been changed.)